### PR TITLE
Fix private methods link

### PR DESCRIPTION
**Pull request by Konrud**
Add `/` to the private methods link to make it work.

### CODE REVIEW

**Review by wbamberg — CHANGES_REQUESTED** [review]
Thank you @Konrud ! Just a typography suggestion.

File: files/en-us/web/javascript/closures/index.md [hunk 1]
```diff
@@ -138,7 +138,7 @@ Run the code using [JSFiddle](https://jsfiddle.net/vnkuZ/7726/).
 
 Languages such as Java allow you to declare methods as private, meaning that they can be called only by other methods in the same class.
 
-JavaScript previously didn't have a native way of declaring [`private methods`](en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields#private_methods), but it was possible to emulate private methods using closures. Private methods aren't just useful for restricting access to code. They also provide a powerful way of managing your global namespace.
+JavaScript previously didn't have a native way of declaring [`private methods`](/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields#private_methods), but it was possible to emulate private methods using closures. Private methods aren't just useful for restricting access to code. They also provide a powerful way of managing your global namespace.
```
[the remark this answers is not in the document]
Comment (Konrud) [reply files/en-us/web/javascript/closures/index.md:141]: Sure. Thank you.

**Review by wbamberg — APPROVED** [review]
👍 thank you!

**Comment by Konrud** [comment]
> Thanks @Konrud for updating this old corner of the docs :)

My pleasure. Thank you.

**Pull request merged**